pkg/validations/base_test.go

```diff
@@ -35,17 +36,24 @@ func newEngine(c config.Config) (validationEngine, error) {
 	return ve, nil
 }
 
+func newCustomCheck() config.Check {
+	return config.Check{
+		Name:        checkName,
+		Description: "some description",
+		Remediation: "some remediation",
```

Comment:
I would make any string we are going to check a constant/variable so if it is changed, it will be changed everywhere that cares.

Reply:
https://github.com/app-sre/deployment-validation-operator/blob/3553f14b6071e509f4e3ba43a13ef4f70746af76/pkg/validations/base_test.go#L24-L29

https://github.com/app-sre/deployment-validation-operator/blob/3553f14b6071e509f4e3ba43a13ef4f70746af76/pkg/validations/base_test.go#L42-L53

https://github.com/app-sre/deployment-validation-operator/blob/3553f14b6071e509f4e3ba43a13ef4f70746af76/pkg/validations/base_test.go#L88-L124

I amended how these strings are handled.